docs/root/intro/version_history.rst

```diff
@@ -6,6 +6,7 @@ Version history
 * access log: added FILTER_STATE :ref:`access log formatters <config_access_log_format>` and gRPC access logger.
 * api: remove all support for v1
 * build: official released binary is now built against libc++.
+* cluster: added aggregate cluster that allows load balancing between clusters.
```

Comment:
Can you `:ref`: link to the docs from here?

Reply:
Done